frontend/workflows/experimentation/src/list-view.tsx

```diff
@@ -22,7 +22,7 @@ const useStyles = makeStyles(theme => ({
   },
   paper: {
     width: "100%",
-    marginBottom: theme.spacing(2),
+    marginBottom: "8px",
```

Comment:
should we remove the theme param above since it's unused now?

Reply:
Actually reverted, seemed necessary before but I must've fixed this somewhere.